mantidproject/mantid
# Indirect System test - OSIRISIqtAndIqtFitMulti

The OSIRISIqtAndIqtFitMulti system test should now pass on Windows and Linux. It is uncertain if this will work on Mac yet but this can be investigated in #11989 

**To test:**
- Run the ISISIndirectInelastic.OSIRISIqtAndIqtFitMulti system test on windows and Ubuntu and ensure it passes. Ensure it passes on the build servers as well
  - _The new reference files should be downloaded automatically when the system tests are run_

Fixes #16515

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
